src/common/playlists/PlayHeaderActions.jsx

```diff
@@ -31,17 +31,21 @@ const PlayHeaderActions = ({ play }) => {
           <span className="sr-only">GitHub</span>
         </a>
       )}
-      {play.blog && (
+      {play.blog || play.path &&
```

Comment:
This check is not required.

- path is mandatory to be there
- blog we are anyway checking in href.
So remove that check `play.blog || play.path &&`

Reply:
Ok

Reply:
@atapas Done